profile/src/v100/entity/persona/persona.rs

```diff
@@ -74,7 +68,7 @@ impl Persona {
     pub fn new(
         persona_creating_factor_instance: HDFactorInstanceIdentityCreation,
         display_name: DisplayName,
-        persona_data: Option<PersonaData>,
+        persona_data: impl Into<Option<PersonaData>>,
```

Comment:
Makes it possible to pass `PersonaData` directly without having to wrap it in a `Some` which I think is very nice (similar to Swift's behaviour).